wallet/src/de/schildbach/wallet/ui/dashpay/PlatformRepo.kt

```diff
@@ -105,30 +112,29 @@ class PlatformRepo(val walletApplication: WalletApplication) {
             val profileDocuments = Profiles(platform).getList(userIds)
             val profileById = profileDocuments.associateBy({ it.userId }, { it })
 
-            // TODO: Replace this Platform call with a query into the local database
-            val toContactDocuments = ContactRequests(platform).get(userId, toUserId = false, retrieveAll = true)
+            val dashPayDocuments = Profiles(platform).getList(userIds)
```

Comment:
this is a duplicate call, need to remove it.

Reply:
https://github.com/dashevo/dash-wallet/pull/430/commits/98db25c7eec687c20178c651d6f8945fbe404ea0